app/models/replication_transfer.rb

```diff
@@ -25,7 +25,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        bag_man_request.cancel!(reason)
+        bag_man_request&.cancel!(reason)
```

Comment:
I'm not familiar with this syntax - is this a typo?  (If it's a typo, the specs need to be improved to ensure this is covered.)